.azure-pipelines/azure-pipelines.yml

```
@@ -214,6 +243,17 @@ stages:
             - name: RHEL 8.8
               test: rhel/8.8
 
+  - stage: Remote_2_16
+    displayName: Remote 2.16
+    dependsOn: []
+    jobs:
+      - template: templates/matrix.yml
+        parameters:
+          testFormat: 2.16/{0}/1
+          targets:
+            - name: RHEL 8.7
+              test: rhel/8.7
```

```suggestion
            - name: RHEL 8.8
              test: rhel/8.8
```
should be 8.8 like in devel